autoscheduler/frontend/src/components/LandingPage/HelpText.tsx

```diff
@@ -3,29 +3,28 @@ import {
   Paper, Typography, Grid, Box,
 } from '@material-ui/core';
 
-const HelpText: React.SFC = function App() {
-  return (
-    <Grid
-      container
-      direction="row"
-      justify="space-around"
-      alignItems="center"
-      style={{ marginTop: '5%', marginBottom: '2%' }}
-    >
-      <Paper style={{ width: '55%', maxWidth: '800px' }}>
+const HelpText: React.SFC = () => (
```

Comment:
I made the function into an arrow function so that it would show up with the proper name in the React DevTools. Before, it would display as `<App>`, but now it shows `<HelpText>`, which is more descriptive.